.eslintrc.js

```diff
@@ -1,10 +1,34 @@
 module.exports = {
+  root: true,
   extends: '@mate-academy/eslint-config',
+  plugins: ['react', 'jest'],
   env: {
-    jest: true
+    es2021: true,
   },
   rules: {
-    'no-proto': 0
+    'no-proto': 0,
   },
-  plugins: ['jest']
+  overrides: [
+    {
+      files: ['src/client/**/*.{js,jsx}'],
+      env: {
+        browser: true,
+      },
+      rules: {
+        'react/jsx-uses-vars': 'error',
+      },
+    },
+    {
+      files: ['src/server/**/*.{js}'],
+      env: {
+        node: true,
+      },
+    },
+    {
+      files: ['**/*.test.{js,jsx,ts,tsx}'],
```

Comment:
Ensure that the Jest environment is correctly set up for all test files. If there are specific Jest rules or plugins you want to apply, consider adding them here.